internal/prometheus/prometheus.go

```diff
@@ -149,6 +149,21 @@ func (p *Provider) Clear() {
 	p.inner.Clear()
 }
 
+// Register register the collectors on OPA prometheus registry
+func (p *Provider) Register(c prometheus.Collector) error {
+	return p.registry.Register(c)
+}
+
+// MustRegister register the collectors on OPA prometheus registry and panics when an error occurs
```

Comment:
Do we need this? I don't think it's used, let's just keep this minimal. 🤔

Reply:
I have implemented these methods here only to implement the interface prometheus.Registerer, If Prometheus already has an interface for registering, I prefer to use that instead of creating a specific for OPA.

Reply:
Ah, I see. Thanks, that makes sense.